src/main/webapp/app/exercises/quiz/manage/short-answer-question/short-answer-question-edit.component.ts

```diff
@@ -208,6 +208,8 @@ export class ShortAnswerQuestionEditComponent implements OnInit, OnChanges, Afte
             this,
         );
         this.changeDetector.detectChanges();
+        this.parseMarkdown(this.questionEditorText);
+        this.questionUpdated.emit();
```

Comment:
This is not necessary since these two lines of code will be only called when the editor is opened and it will not be called if the editor's content has changed.